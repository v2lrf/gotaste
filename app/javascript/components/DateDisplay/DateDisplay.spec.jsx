import React from 'react'
import { DateDisplay } from '.'

describe('DateDisplay', () => {
  let component
  let props
  beforeEach(() => {
    props = {
      timestamp: 1546627341836
    }

    component = shallow(<DateDisplay {...props} />)
  })

  describe('Snapshots', () => {
    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
