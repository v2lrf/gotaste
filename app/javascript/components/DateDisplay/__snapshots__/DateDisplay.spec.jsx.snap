// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateDisplay Snapshots should match the snapshot 1`] = `
<time
  className="inline-block bg-white text-center px-2 py-3 h-16 w-16 rounded shadow"
  dateTime={1546627341836}
>
  <span
    className="block text-red-darker font-bold text-2xl"
  >
    4
  </span>
  <span
    className="block text-grey-darker text-xs font-bold uppercase"
  >
    Jan
  </span>
</time>
`;
